pkgs/development/compilers/cudatoolkit/common.nix

```diff
@@ -67,13 +72,21 @@ backendStdenv.mkDerivation rec {
   nativeBuildInputs = [
     perl
     makeWrapper
+    rsync
     addOpenGLRunpath
     autoPatchelfHook
     autoAddOpenGLRunpathHook
   ] ++ lib.optionals (lib.versionOlder version "11") [
     libsForQt5.wrapQtAppsHook
+  ] ++ lib.optionals (lib.versionAtLeast version "11.8") [
+    qt6Packages.wrapQtAppsHook
   ];
-  buildInputs = [
+  buildInputs =
+    lib.optionals (lib.versionOlder version "11") [
```

Comment:
```suggestion
  buildInputs = lib.optionals (lib.versionOlder version "11") [
```

Reply:
All of nixpkgs using nixpkgs-fmt so that I don't have to stop myself running `<Space>,f` prior to `:Gw` - WHEEEEEN